python/tvm/relay/backend/utils.py

```diff
@@ -15,6 +15,13 @@
 # specific language governing permissions and limitations
 # under the License.
 """Utility backend functions."""
+from enum import Enum
+
+
+class CallType(Enum):
```

Comment:
just speculating: is it possible to get these values from the enum in c++ in any way? Just thinking there's the possibility that changes to the enum in c++ might not be reflected here

Reply:
If it is I'm not sure how :P Agreed there's a drifting risk although hopefully the tests would detect this.

Reply:
Yeah I didn't have anything in mind either :)